### ISSUE TITLE
Any plan to wrap it in an asciidoctorj bundle?

### ISSUE THREAD

**Issue by rmannibucau** [asciidoctor/asciidoctor-chart#43]
Goal would be to use it directly with asciidoctorj and not have to clone the gem in a dedicated folder.
Guess it can be part of the repo release?

**Comment by ggrossetie** [comment]
Java binding is typically (i.e., in the Asciidoctor community) done in a dedicated repository: 

- https://github.com/asciidoctor/asciidoctorj-reveal.js
- https://github.com/asciidoctor/asciidoctorj-diagram
- https://github.com/asciidoctor/asciidoctorj-chart 👈🏻 

I believe @uniqueck is working on it...

**Comment by uniqueck** [comment]
Yes indeed. For now we have to release it manually. A new version of asciidoctor-chart is available, so we can release the wrapped version. @Mogztter how wo we like to test the process and the interaction between these both repositories?

**Comment by uniqueck** [comment]
@rmannibucau  version 1.0.0 is released. https://search.maven.org/artifact/org.asciidoctor/asciidoctorj-chart/1.0.0/jar
